common/test/db_cache/bok_choy_schema.sql

```diff
@@ -2528,20 +2528,6 @@ CREATE TABLE `programs_programsapiconfig` (
   CONSTRAINT `programs_programsa_changed_by_id_b7c3b49d5c0dcd3_fk_auth_user_id` FOREIGN KEY (`changed_by_id`) REFERENCES `auth_user` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
-DROP TABLE IF EXISTS `psychometrics_psychometricdata`;
```

Comment:
Yeah again I'd have to defer to @symbolist or @macdiesel . (Also curious what the answer is.)

Thanks for even thinking of the DB cruft; glad to clean that up as part of this.